src/Plugin/GraphQL/Schemas/SchemaPluginBase.php

```diff
@@ -26,6 +27,9 @@
 use Psr\Log\LoggerInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
+/**
+ *
+ */
```

Comment:
This mutes the error but it doesn't fix it.

Reply:
I understand. I have already removed these blank comments. These changes have been automatically included by the "phpcbf" tool.